keywords.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/local/bin/python3
+# tested on debian 8.5, you need python-pip to install the following modules:
+# pyyaml, bs4, slackclient, translate
+
+from random import randint
+import random
+from slack_commands import __send_response, __reaction
+
+
+def keywords(sc, evt, yamldata, count):
+    if "channel" in evt and evt["type"] == "message":
+        message = evt["text"].strip()
+        if "bot_id" not in evt:
```

Comment:
If you're not opposed to this style, short-circuiting
```
if "bot_id" in evt:
  return count

<rest of the stuff
```
would reduce indentation level.

Reply:
this was a great suggestion, it removed a ton of redundant code too. Thanks!